src/main/java/com/hackaton/model/ResultsModel.java

```diff
@@ -10,18 +10,7 @@
     private List<ResultInterface> data;
     private final Object[] COLUMNS = {"Nazwa", "Ilość"};
 
-    public ResultsModel() {
-        //TODO usunąć konstruktor
-        data = new ArrayList<>();
-        data.add(new TemporaryModelClass("start", 1));
-        data.add(new TemporaryModelClass("dwa", 5));
-        data.add(new TemporaryModelClass("trzy", 8));
-        data.add(new TemporaryModelClass("cztery", 5));
-//        for(int i=0; i<50;i++){
-//            data.add(new TemporaryModelClass("aaa", 0));
-//        }
-//        data.add(new TemporaryModelClass("koniec", 9));
-    }
+    public ResultsModel(){}
```

Comment:
Czy ten konstruktor jest nadal potrzebny?

Reply:
Niech se będzie. Może do czegoś się nam jeszcze przyda. Nikomu nie wadzi na razie

Reply:
Ja bym go wywalił. Napisanie konstruktora to 3 wciśnięcia klawiszy w IntelliJ